integration-app/src/main/java/uk/gov/crowncommercial/esourcing/integration/app/SalesforceConfiguration.java

```diff
@@ -32,4 +179,71 @@ public RfxStatusListApi rfxStatusApi(ApiClient apiClient) {
     rfxStatusListApi.setApiClient(apiClient);
     return rfxStatusListApi;
   }
+
+  private ExchangeFilterFunction requestPasswordTokenRequestProcessor() {
+    return ExchangeFilterFunction.ofRequestProcessor(
+        request ->
+            Mono.just(
+                ClientRequest.from(request)
+                    .body(
+                        ((BodyInserters.FormInserter<String>) request.body())
```

Comment:
In Eclipse this is the only warning there is in the code so it stands out. Not ideal to cast without checking. If we can check the type before casting then that's something. Not sure what else to do about it. I had a go at doing this and got nowhere - it's too "lambda/mono"y for my liking.

Reply:
IntelliJ flagged this as well, wasn't sure how best to change it